Source/Core/Duality/Resources/Scene.cs

```diff
@@ -491,9 +491,9 @@ internal void Update()
 
 			// Update low fps physics state
 			if (!physicsLowFps)
-				physicsLowFps = Time.LastDelta > Time.MsPFMult && physTime > Time.LastDelta * 0.85f;
+				physicsLowFps = Time.DeltaTime > Time.MillisecondsPerFrame && physTime > Time.DeltaTime * 0.85f;
```

Comment:
Do you think if changing `LastDelta` to `DeltaTime` is sufficient in the `physicsLowFps` test?